virttest/virt_vm.py

```diff
@@ -666,6 +666,8 @@ def _get_mac_addr(nic_index):
             mac = None
             try:
                 mac = self.get_mac_address(nic_index)
+                if not mac:
+                    raise VMMACAddressMissingError(nic_index)
```

Comment:
'get_mac_address' will not raise this exception and return None? then the logic should be fixed, but not using try.except to control the workflow